src/signals/settings/categories/components/IconInput/IconInput.tsx

```diff
@@ -123,20 +128,21 @@ export const IconInput = ({ formMethods, icon }: Props) => {
   return (
     <FieldGroup>
       <StyledHeading>Icoon</StyledHeading>
-
-      {/* Informatie */}
-
+      <Detail
+        header={'Het icoon wordt getoond op de openbare meldingenkaart.'}
+        content={'Zorg voor een circel en exporteer als SVG.'}
```

Comment:
cirkel right

Reply:
![image](https://github.com/Amsterdam/signals-frontend/assets/25867841/e5d021eb-ad0a-43a5-9b4d-c604f9fc6e21)
 Is gefixt